mne/coreg.py

```diff
@@ -77,9 +77,8 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
 
     Parameters
     ----------
-    mne_root : None | str
-        The mne root directory (only needed if MNE_ROOT is not specified as
-        environment variable).
+    mne_root : None
+        This argument is not used anymore.
```

Comment:
Yeah that was also my concern, although `create_default_subject()` is probably rarely used programmatically...